tests/network/network_connectivity_test.go

```diff
@@ -0,0 +1,70 @@
+package network_test
+
+import (
+	"flag"
+	"fmt"
+
+	"github.com/google/goexpect"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"kubevirt.io/kubevirt/pkg/api/v1"
+	"kubevirt.io/kubevirt/pkg/kubecli"
+	"kubevirt.io/kubevirt/tests"
+)
+
+var _ = Describe("Network Connectivity", func() {
+
+	flag.Parse()
+	virtClient, err := kubecli.GetKubevirtClient()
+	tests.PanicOnError(err)
+
+	var vma *v1.VirtualMachineInstance
```

Comment:
Please use arrays to store the VMs structs, and then replace the duplicate code with loops.